aci/data_source_aci_fvnsmcastaddrinstp.go

```diff
@@ -0,0 +1,75 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/client"
+	"github.com/ciscoecosystem/aci-go-client/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAciMulticastAddressPool() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   dataSourceAciMulticastAddressPoolRead,
+		SchemaVersion: 1,
+		Schema: AppendAttrSchemas(map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"multicast_address_block": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: AppendAttrSchemas(map[string]*schema.Schema{
+						"dn": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"from": {
+							Type:     schema.TypeString,
+							Required: true,
```

Comment:
I thought we put all things to optional on data sources.

Reply:
ok